src/Components/index/index.js

```diff
@@ -1,7 +1,8 @@
 import { Navbar } from '../Navbar/Navbar';
 import { Footer } from '../Footer/Footer';
 import { Button } from '../Buttons/Button';
+import { TextBadgeSquare } from '../TextSquareBadge/TextSquareBadge';
+import { Ratings } from '../Ratings/Ratings';
```

Comment:
suggestion: instead of import and then export, you can directly export from this file itself like 
export { Footer } from '../Footer/Footer';